processes/chicagoISAC.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+import requests
+from requests.exceptions import HTTPError, ConnectionError
+
+from .core import CoreProcess
+from mappings.core import MappingError
+from mappings.chicagoISAC import ChicagoISACMapping
+from managers import WebpubManifest
+from logger import createLog
+
+logger = createLog(__name__)
+
+class ChicagoISACProcess(CoreProcess):
+
+    def __init__(self, *args):
+        super(ChicagoISACProcess, self).__init__(*args[:4])
+
+        self.ingestOffset = int(args[5] or 0)
+        self.ingestLimit = (int(args[4]) + self.ingestOffset) if args[4] else 5000
+        self.fullImport = self.process == 'complete' 
+
+        # Connect to database
+        self.generateEngine()
+        self.createSession()
+
+        # S3 Configuration
+        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.createS3Client()
+
+    def runProcess(self):
+        with open('chicagoISAC_metadata.json') as f:
+                chicagoISACData = json.load(f)
+
+        for metaDict in chicagoISACData:
+            self.processChicagoISACRecord(metaDict)
+
+        self.saveRecords()
+        self.commitChanges()
+
+    def processChicagoISACRecord(self, record):
+        try:
+            chicagoISACRec = ChicagoISACMapping(record)
+            chicagoISACRec.applyMapping()
+            self.storePDFManifest(chicagoISACRec.record)
+            self.addDCDWToUpdateList(chicagoISACRec)
+            
+        except (MappingError, HTTPError, ConnectionError, IndexError, TypeError) as e:
+            logger.debug(e)
```

Comment:
Could be good to do `logger.exception` here, which will log the stack trace for you